vizro-core/src/vizro/static/css/accordion.css

```diff
@@ -42,35 +42,37 @@
   overflow-anchor: none;
   background-color: inherit;
   letter-spacing: -0.014px;
-  padding-left: 0;
+  padding-top: var(--spacing-04);
```

Comment:
If you add the padding here, can you remove the gap in the accordion class? Otherwise the space above and below the accordion title is not the same (see your screenshot on SUMMARY). Or is this intended?

Reply:
Apparently, it's intended given the designs - if we could just incorporate the hover color as in the designs, then I think we should be good